File: minseon/23_01_09_01.py
#부족한 금액 계산하기

def solution(price, money, count):
    answer = -1
    inpl = 0
    for a in range(1,count+1):
        inpl += price * a
        print(inpl)
        # 내가 이용한 놀이기구의 가격

        if inpl > money:
            answer = inpl - money
        else:
            answer = 0

    return answer

File: seungyeop/23_01_09_01.java
class Solution {
    public long solution(int price, int money, int count) {
        long cost = 0;  // 비용이 상당히 클 경우 int로 다 안 받아짐
        for (int i = 1; i <= count; i++) {
            cost += price * i;
        }
        long answer = 0;
        if (cost - money > 0) {
            answer = cost - money;
        }
        return answer;
    }
}
